Dockerfile

```diff
@@ -3,6 +3,7 @@
 # Copyright (C) 2020 Bosch Software Innovations GmbH
 # Copyright (C) 2021-2022 Bosch.IO GmbH
 # Copyright (C) 2021 Alliander N.V.
+# Copyright (C) 2021-2022 BMW CarIT GmbH
```

Comment:
> These "components" are installed in separate layers to benefit from Docker's caching if a
tool version changes, but others stay fixed.

I believe we should improve this bit of my own wording as say:

These "components" are installed in separate multi-images to benefit from Docker's parallel building of those, and the tool binaries are then copied to the final image. In contrast to a layered approach this avoids rebuilding if only a single tool version changes.